### PR TITLE
Readable COCO and datumaro format for CJK

### LINKED ISSUE #231: Readable COCO and datumaro format for CJK
When a dataset with categories in CJK (e.g., "고양이", "ネコ", "猫") is converted to COCO or datumaro and writed, the categories are saved in Unicode as below.
```json
"categories": [{
    "id": 1,
    "name": "\uace0\uc591\uc774",
    "supercategory": ""
}, {
    "id": 2,
    "name": "\u30cd\u30b3",
    "supercategory": ""
}, {
    "id": 3,
    "name": "\u732b",
    "supercategory": ""
}],
```
There is no problem when loading this file, but I feel inconvenient when reading the json file directly.

I think this would be done by adding `ensuer_ascii = False` to `json.dump` in https://github.com/openvinotoolkit/datumaro/blob/ec4b013308534681d23ae3720dde7cdb844aed21/datumaro/plugins/coco_format/converter.py#L98-L99 and https://github.com/openvinotoolkit/datumaro/blob/ec4b013308534681d23ae3720dde7cdb844aed21/datumaro/plugins/datumaro_format/converter.py#L103-L104

Thank you.

### PR DESCRIPTION
**Pull request by jaesuny**
### Summary
Resolve #231.

### Checklist

- [x] I submit my changes into the `develop` branch
- [ ] I have added description of my changes into [CHANGELOG](https://github.com/openvinotoolkit/datumaro/blob/develop/CHANGELOG.md)
- [ ] I have updated the [documentation](
  https://github.com/openvinotoolkit/datumaro/tree/develop/docs) accordingly
- [x] I have added tests to cover my changes
- [x] I have [linked related issues](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword)

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/openvinotoolkit/datumaro/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Review by zhiltsov-max — CHANGES_REQUESTED** [review]
Thank you for the PR! Please, add a line (`DATUM_231 = ...`) in `tests/requirements.py` and refer to it in the test annotations.

File: tests/test_coco_format.py [hunk 1]
```diff
@@ -718,6 +718,36 @@ def test_can_save_and_load_images(self):
             self._test_save_and_load(expected_dataset,
                 CocoImageInfoConverter.convert, test_dir)
 
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
```
Comment (zhiltsov-max) [inline tests/test_coco_format.py:721]:
```suggestion
    @mark_requirement(Requirements.DATUM_231)
```

File: tests/test_datumaro_format.py [hunk 2]
```diff
@@ -111,6 +111,37 @@ def test_relative_paths(self):
             self._test_save_and_load(test_dataset,
                 partial(DatumaroConverter.convert, save_images=True), test_dir)
 
+
+    @mark_requirement(Requirements.DATUM_GENERAL_REQ)
```
Comment (zhiltsov-max) [inline tests/test_datumaro_format.py:115]:
```suggestion
    @mark_requirement(Requirements.DATUM_231)
```

**Comment by jaesuny** [comment]
> Thank you for the PR! Please, add a line (`DATUM_231 = ...`) in `tests/requirements.py` and refer to it in the test annotations.

Updated!

File: datumaro/plugins/coco_format/converter.py [hunk 3]
```diff
@@ -462,7 +462,7 @@ class _StuffConverter(_InstancesConverter):
 class _PanopticConverter(_TaskConverter):
     def write(self, path):
         with open(path, 'w') as outfile:
```
Comment (zhiltsov-max) [inline datumaro/plugins/coco_format/converter.py:464]:
```suggestion
        with open(path, 'w', encoding='utf-8') as outfile:
```

File: datumaro/plugins/datumaro_format/converter.py [hunk 4]
```diff
@@ -101,7 +101,7 @@ def write_categories(self, categories):
 
     def write(self, save_dir):
         with open(osp.join(save_dir, '%s.json' % self._name), 'w') as f:
```
Comment (zhiltsov-max) [inline datumaro/plugins/datumaro_format/converter.py:103]:
```suggestion
        with open(osp.join(save_dir, '%s.json' % self._name), 'w',
                encoding='utf-8') as f:
```

File: datumaro/plugins/coco_format/converter.py [hunk 5]
```diff
@@ -101,7 +101,7 @@ def write(self, path):
                 next_id += 1
 
         with open(path, 'w') as outfile:
```
Comment (zhiltsov-max) [inline datumaro/plugins/coco_format/converter.py:103]:
```suggestion
        with open(path, 'w', encoding='utf-8') as outfile:
```

**Review by zhiltsov-max — APPROVED** [review]

**Pull request merged**